Makefile

```diff
@@ -24,7 +24,7 @@ BUILD_BATS_MOCK_VERSION ?= 2f9811faf43593ad7b59a0f245d8807b776e5072
 BUILD_BATS_SUPPORT_VERSION ?= 004e707638eedd62e0481e8cdc9223ad471f12ee
 DOCKER_IMAGE_BATS := cftk/bats
 # DOCKER_TAG_BATS is the image semver and has no correlation to bats versions
-DOCKER_TAG_BATS ?= 0.5.0
+DOCKER_TAG_BATS ?= 0.6.0
```

Comment:
Assembling the `content` output from fragments requires terraform in the docker image.  It was not previously there.

@Jberlinsky What's the process for getting a new docker image in place?

Reply:
Nevermind, I see this is handled in `terraform-google-startup-scripts.yml` `make docker_build_bats`